### PR TITLE
Add instructions for tile authors to use secure binding credentials

### PR DESCRIPTION
**Pull request by nodo**
[#154130321]

Signed-off-by: Henry Stanley <hstanley@pivotal.io>

### CODE REVIEW

**Comment by nodo** [comment]
@avade - if you're happy with this, we'll add PRs for 0.18.0 and 0.19.0 branches.

**Comment by nodo** [comment]
Please, when this PR is ready to be merged, could you merge it to 0.18.0 and 0.19.0 as well?

File: tile.html.md.erb [hunk 1]
```diff
@@ -200,3 +200,30 @@ Pre-delete:
...
+To allow users to enable and disable this feature in the Ops Manager UI, add an element to the `property_blueprints` section of your tile's `handcraft.yml`, and add a selector which templates in the appropriate manifest snippet.
+
+For an example, see the **example-kafka-on-demand-tile**:
+
+* ['Enable Secure Bindings' form field](https://github.com/pivotal-cf-experimental/example-kafka-on-demand-tile/blob/e206e04a1eb80a5f53a5edd9a1f11e096bea5f4f/metadata_parts/handcraft.yml#L39-L46)
+* [`property_blueprint`](https://github.com/pivotal-cf-experimental/example-kafka-on-demand-tile/blob/e206e04a1eb80a5f53a5edd9a1f11e096bea5f4f/metadata_parts/handcraft.yml#L605-L632)
+* [broker manifest template](https://github.com/pivotal-cf-experimental/example-kafka-on-demand-tile/blob/e206e04a1eb80a5f53a5edd9a1f11e096bea5f4f/metadata_parts/handcraft.yml#L112)
+
+
+<p class="note">In order to use the secure binding credentials feature you must use PCF 1.11 or later.</p>
```
Comment (avade) [inline tile.html.md.erb]: This should be:

<p class="note"> PCF 2.0 or later is required in order for your tile to use the secure binding credentials feature</p>

**Comment by avade** [comment]
@ghanna2017 PR is ready to be reviewed/merged, could you merge it to 0.18.0 and 0.19.0 as well?

**Comment by ghanna2017** [comment]
Thanks @nodo @avade . I'll merge to .18/.19 also.

**Comment by jncd** [comment]
@avade and @henryaj 
Just to confirm, this is the "runtime CredHub" and not the "BOSH CredHub"?
Because there are two distinct CredHubs, we want to disambiguate on first us of "CredHub" on a page. 
screen shot 2018-01-16 at 10 45 09 am

**Comment by henryaj** [comment]
Correct @jncd - this is the runtime CredHub. 👍

**Pull request merged**